lib/ansible/config/manager.py

```diff
@@ -152,7 +153,13 @@ def find_ini_config_file():
         if os.path.isdir(path0):
             path0 += "/ansible.cfg"
     try:
-        path1 = os.getcwd() + "/ansible.cfg"
+        path1 = os.getcwd()
+        perms1 = os.stat(path1)
+        if perms1.st_mode & stat.S_IWOTH:
+            # Ansible is in a world writable directory, ignoring it as ansible.cfg source.
+            path1 = None
```

Comment:
Do we want to do a display.warning() here?

Reply:
@bcoca says that we don't have a display object created yet.  If we attempt to use a display object, it will throw an error.  I think we should add a commented out call to display.warn() here so that someone can remember to add this once we're able to.  (I'm planning to replace/reimplement the backend of display with logging infrastructure for 2.8.  And once we have that, we can either queue early updates for later logging or we can push the earliest messages to a log that does not depend on configuration.)